supplementary_style_guide/style_guidelines/structure.adoc

```diff
@@ -1,6 +1,66 @@
 [[structure]]
 = Structure
 
+[[abstracts-or-shortdesc]]
+== Abstracts or short descriptions
+
+To help readers find the information that they need or to confirm that they are in the right place, every module and assembly must include a short description, also called an abstract.
+
+This description is usually at least 2-3 sentences long, and you can scan it in a few seconds. It exists between the title and the main content, connecting them and providing context and disambiguation.
+
+Follow these guidelines when writing short descriptions:
+
+[discrete]
+=== Content
+
+* Include user intent:
+** *What* users must do. This content is similar to what is in the title but should not just repeat the same information.
+** *Why* users must complete an action. You must build from the information in the title. This content helps users understand why completing an action is important or beneficial to them.
+*** If the title of your module is “Creating a Kafka instance in OpenShift Streams for Apache Kafka”, your short description could begin something like this: “To connect OpenShift Streams for Apache Kafka to other applications or services [*why*], create and configure a Kafka instance in the web console [*what*].”
```

Comment:
Yeah I think that either this example here needs to be removed completely, or it needs to be formatted a little differently. And listed as an example. Right now, it's shown as a bullet under the "Why" item, but it's really applicable to both. If we keep it, could maybe instead format it as normal text under the main bullet. Could do something like

```
* Include user intent:
+
--
** *What* users must do. This content is similar to what is in the title but should not just repeat the same information.
** *Why* users must complete an action. You must build from the information in the title. This content helps users understand why completing an action is important or beneficial to them.
--
+
For example, if the title of your module is “Creating a Kafka instance in OpenShift Streams for Apache Kafka”, your short description could begin something like this: “To connect OpenShift Streams for Apache Kafka to other applications or services [*why*], create and configure a Kafka instance in the web console [*what*].”
```

That would format it to look something like:
![image](https://user-images.githubusercontent.com/7443887/226415643-48e97fa3-d3a0-44af-a339-8d18053cc6f1.png)